Cargo.toml

```diff
@@ -54,14 +54,19 @@ windows = { version = "0", optional = true }
 
 [build-dependencies]
 # Bindgen 0.70.0 and later cause build failures when compiling to WASM. For more details, see:
-# 
+#
```

Comment:
I think there was meant to be a link to https://github.com/ajrcarey/pdfium-render/issues/156 here. I'll add it separately.